akka-http-tests/src/multi-jvm/scala/akka/http/AkkaHttpServerLatencyMultiNodeSpec.scala

```diff
@@ -176,12 +176,12 @@ class AkkaHttpServerLatencyMultiNodeSpec extends MultiNodeSpec(AkkaHttpServerLat
         enterBarrier("load-gen-ready")
 
         runOn(server) {
-          val (_, port) = SocketUtil.temporaryServerHostnameAndPort()
+          val port = 0
           info(s"Binding Akka HTTP Server to port: $port @ ${myself}")
-          val futureBinding = Http().newServerAt("0.0.0.0", port).bind(routes)
+          val binding = Http().newServerAt("0.0.0.0", port).bind(routes).futureValue
 
-          _binding = Some(futureBinding.futureValue)
```

Comment:
oh I didn't notice before this was already here - looks like a safe change then :+1: